tests/plugins/test_koji_import.py

```diff
@@ -1702,7 +1702,9 @@ def test_koji_import_set_manifest_list_info(self, caplog, tmpdir, monkeypatch, o
                                             session=session,
                                             docker_registry=True,
                                             add_tag_conf_primaries=not is_scratch)
-        group_manifest_result = {'myproject/hello-world': digest} if digest else {}
+        group_manifest_result = {}
+        if digest:
+            group_manifest_result = {'manifest_digest': ManifestDigest(v2_list=digest)}
```

Comment:
```suggestion
            group_manifest_result = {'manifest_digest': digest}
```

`digest` is already a `ManifestDigest` instance.

Reply:
Still needs fixing.

Reply:
@mlangsdorf, please take a look at this.

Reply:
Fixed, finally.